app/models/user/email.rb

```diff
@@ -0,0 +1,36 @@
+# Represents an email address belonging to a user.
+class User::Email < ActiveRecord::Base
```

Comment:
@lowjoel I've updated here, not sure if I should override `destroy!`, but cannot find a better name other than `destroy!`